.github/workflows/deploy.yml

```diff
@@ -190,3 +207,25 @@ jobs:
       project_type: ui
       azure_app_name: ${{ matrix.azure_app_name }}
     secrets: inherit
+
+  deploy-functions:
+    name: ${{ matrix.job_name }}
+    needs: [ deploy-ui-services ]
```

Comment:
question: I think it might be better to do `needs: [ build-projects-upload-artifacts ]` for this and deploy the function(s) in parallel with the APIs, or have it depend on the API deployment step and deploy the function(s) and UIs in parallel.